src/widget/tabs/camera-tab.js

```diff
@@ -270,7 +270,7 @@ class CameraTab {
     return canvasToBlob(canvas, 'image/jpeg', 0.9, blob => {
       canvas.width = canvas.height = 1
       blob.name = 'camera.jpg'
-      this.dialogApi.addData('object', blob)
+      this.dialogApi.addData('object', [blob])
```

Comment:
Why `blob` is wrapped into array? Didn't noticed changes to the `addData`

Reply:
because `addData` accept array